css/openhab.css

```diff
@@ -24,13 +24,23 @@ li.search:hover {
   border: none !important; 
   transition: 0.2s ease-in;
   display: block;
-  margin: 17px 0 0 0;
+  margin: 17px 0 0 0 !important;
 }
 
-.search-form ::-webkit-input-placeholder,
-.search-form ::-moz-placeholder,
-.search-form ::-ms-input-placeholder {
-    color: #fff;
+.sticky .search-form-input {
+  margin: 9px 0 0 0 !important;
+}
+
+::-webkit-input-placeholder {
```

Comment:
Using this without class or id is dangerous. There is at least one other input box that already has a white background, e.g. the input box that appears on mobile devices (id is querymob). 

Same is true for the other placeholder pseudo elements.